.github/workflows/ci-frontend-e2e.yml

```diff
@@ -50,15 +53,75 @@ jobs:
           ../api/bin/wait-for-api.sh
         shell: bash
 
-      - name: Run E2E Tests
+      - name: Build a prod version of the site
         run: |
-          npm run build
           cat .env.development >> .env.local
-          npm run test:e2e
+          npm run build
```

Comment:
it looks like since the sharded test runs are all going in parallel that the npm and playwright setup steps are not able to be cached, so 5 mins of processing time is being spent on each run on something that it seems like could be shared. I'm a novice with github actions, but is this a place where a reusable workflow could help? https://docs.github.com/en/actions/sharing-automations/reusing-workflows#creating-a-reusable-workflow

Reply:
Yes, this would be a good follow up.

Reply:
I created ticket https://github.com/HHS/simpler-grants-gov/issues/2813